build.gradle

```diff
@@ -189,11 +189,14 @@ dependencies {
   compile group: 'uk.gov.hmcts.reform', name: 'java-logging', version: versions.reformLogging
   compile group: 'uk.gov.hmcts.reform', name: 'java-logging-appinsights', version: versions.reformLogging
   compile group: 'uk.gov.hmcts.reform', name: 'service-auth-provider-client', version: '3.0.0'
-  compile group: 'uk.gov.hmcts.reform', name: 'sscs-common', version: '3.0.74'
+  compile group: 'uk.gov.hmcts.reform', name: 'sscs-common', version: '3.0.75'
 
   //Remove when our dependencies pull in this version or later
-  compile 'com.fasterxml.jackson.core:jackson-databind:2.9.9.3'
-  integrationTestCompile 'com.fasterxml.jackson.core:jackson-databind:2.9.9.3'
+  compile group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.9.10'
+  integrationTestCompile 'com.fasterxml.jackson.core:jackson-databind:2.9.10'
+
+  compile group: 'com.microsoft.azure', name: 'applicationinsights-logging-logback', version: '2.4.1'
+  compile group: 'com.microsoft.azure', name: 'applicationinsights-spring-boot-starter', version: '2.4.1'
```

Comment:
perhaps want to consider to bump agent too? this will require agent jar and `web.xml`